pkg/operator/ceph/cluster/nodedaemon/reconcile.go

```diff
@@ -234,20 +258,32 @@ func (r *ReconcileNode) removeDisabledNodeDaemons(spec cephv1.ClusterSpec, names
 	return allDisabled
 }
 
+func (r *ReconcileNode) removeDisabledCephExporterDaemons(spec cephv1.ClusterSpec, namespace string) bool {
+	// If the ceph-exporter daemons are disabled in the spec let's remove them
+	allDisabled := true
```

Comment:
There is only one exporter service, so how about if we simplify this by returning:
```go
return spec.CrashCollector.Disable
```
A similar simplification could be done for the crash collector